app/models/vat.rb

```diff
@@ -0,0 +1,33 @@
+module Shop
+  module Models
+    class VAT
+      class UnknownCategoryError < StandardError; end
```

Comment:
change everywhere class definition to UnknownCategoryError = Class.new(StandardError)

Not a place to explain in details why it's better, I will try to find blog post